### PR TITLE
Adds Sani'Orios as an away site

### PR DESCRIPTION
**Pull request by Alberyk**
This pr adds the asteroid belt of one of the Tajaran planets as a visitable away site. This is content for the next sector we are moving in.

### CODE REVIEW

**Review by JohnWildkins — CHANGES_REQUESTED** [review]
roundstart active atmos processing

```## UNIT_TEST ##: !!! FAILURE !!! [ZAS: Roundstart Active Edges]: 3 edges active at round-start!
## UNIT_TEST ##: ---- Zone 1718 and space (164, 226, 20) have mismatched gas mixtures!
## UNIT_TEST ##: -------- Problem turfs: The plating (164, 225, 20) 
## UNIT_TEST ##: ---- Zone 1718 and ash (163, 225, 20) have mismatched gas mixtures!
## UNIT_TEST ##: -------- Problem turfs: The plating (164, 225, 20) 
## UNIT_TEST ##: ---- Zone 1722 and space (168, 226, 20) have mismatched gas mixtures!
## UNIT_TEST ##: -------- Problem turfs: The plating (168, 225, 20) ```

File: maps/away/away_site/tajara/saniorios_smuggler/saniorios_smuggler.dm [hunk 1]
```diff
@@ -0,0 +1,51 @@
+/datum/map_template/ruin/away_site/saniorios_smuggler
+	name = "Sani'Orios"
+	description = "A gas giant composed of ammonia, Its planetary ring is home to several spaceship wrecks and hidden smuggler bases."
```
Comment (SleepyGemmy) [inline maps/away/away_site/tajara/saniorios_smuggler/saniorios_smuggler.dm:3]:
```suggestion
	description = "A gas giant composed of ammonia. Its planetary ring is home to several spaceship wrecks and hidden smuggler bases."
```

File: maps/away/away_site/tajara/saniorios_smuggler/saniorios_smuggler.dm [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+	suffixes = list("away_site/tajara/saniorios_smuggler/saniorios_smuggler.dmm")
+	sectors = list(SECTOR_SRANDMARR)
+	spawn_weight = 1
+	spawn_cost = 2
+	id = "saniorios_smuggler"
+
+/singleton/submap_archetype/saniorios_smuggler
+	map = "Sani'Orios"
+	descriptor = "A gas giant composed of ammonia, Its planetary ring is home to several spaceship wrecks and hidden smuggler bases."
```
Comment (SleepyGemmy) [inline maps/away/away_site/tajara/saniorios_smuggler/saniorios_smuggler.dm:12]:
```suggestion
	descriptor = "A gas giant composed of ammonia. Its planetary ring is home to several spaceship wrecks and hidden smuggler bases."
```

File: maps/away/away_site/tajara/saniorios_smuggler/saniorios_smuggler.dm [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+/obj/effect/overmap/visitable/sector/saniorios_smuggler
+	name = "Sani'Orios"
+	desc = "A gas giant composed of ammonia, Its planetary ring is home to several spaceship wrecks and hidden smuggler bases."
```
Comment (SleepyGemmy) [inline maps/away/away_site/tajara/saniorios_smuggler/saniorios_smuggler.dm:16]:
```suggestion
	desc = "A gas giant composed of ammonia. Its planetary ring is home to several spaceship wrecks and hidden smuggler bases."
```

File: maps/away/away_site/tajara/saniorios_smuggler/saniorios_smuggler.dm [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+	icon_state = "globe"
+	color = COLOR_DARK_BLUE_GRAY
+
+/obj/effect/overmap/visitable/sector/saniorios_smuggler/get_skybox_representation()
+
```
Comment (SleepyGemmy) [inline maps/away/away_site/tajara/saniorios_smuggler/saniorios_smuggler.dm:21]:
```suggestion
```

**Review by SleepyGemmy — APPROVED** [review]

**Pull request merged**